### ISSUE TITLE
Wrong location reporting

### ISSUE THREAD

**Issue by Botev** [AtomLinter/linter-rust#18]
I've just installed as a brand new both Atom and linter-rust. For some reason all of my errors get reported as line 1, col 1. Would  be nice if that can be fixed.

**Comment by colin-kiegel** [comment]
@Botev: I had that issue, too. But Updating to 0.2.0 and restarting atom today solved it for me. Does it still occur with the most recent version?

**Comment by b52** [comment]
Related to #21 and should be fixed by #25.

**Comment by Botev** [comment]
Sorry for the late reply, was on vacation. The update fixes the problem for most cases but for a block after a match it still is not working. Here is an example:
```rust
match {
		// Code which returns Result<(),String>
		} { // First bracket here should not be indented
			Ok(()) => (),
			Err(msg) => return Err(msg)
		}
```

**Comment by colin-kiegel** [comment]
I don't see how indentation of code could be a problem of linter at all. I think this issue should be filed elsewhere - maybe language-rust or whatever Handels indentation. But I am quite sure it's not linter.

**Comment by b52** [comment]
@Botev Could you provide a short example I could test with?

**Comment by Botev** [comment]
Again sorry for the late replies, but I'm extremely busy lately. Here is a full example (please ignore the logic simplicity):
```rust
fn main(){
    let a = 2;
    match {
        if a == 2{
            Ok(())
        }
        else {
            Err("its not two")
        }
        } { // This should be indent one level back so to start at the same indent level as 'match'
            Ok(_) => println!("It is ok"),
            Err(msg) => println!("ooops: {}", msg)
        }
    } // As a consequence this also not indented correctly
```

**Comment by b52** [comment]
@Botev That code doesn't contain a problem, i.e., error or warning, that could be indicated by the linter. Even if I manually add a code to provoke a warning, it is correctly placed and labeled, as one can see in the attached screenshot.

![untitled](https://cloud.githubusercontent.com/assets/23704/8761432/ce1218f6-2d51-11e5-967d-c480930d55f5.png)

So, I'm in favor of closing this issue.

**Comment by colin-kiegel** [comment]
@Botex: See my [comment above](https://github.com/AtomLinter/linter-rust/issues/18#issuecomment-119946412). I think you should file these kinds of issues with automatic indentation in language-rust (e.g. take a look at https://github.com/zargony/atom-language-rust/issues/14). ;-)

**Comment by Botev** [comment]
My, bad jeez, I for some reason thought this is the issue about the indention! Please take my sincere apologies, having been working to too late, lately. 

Btw with the update the error gets reported at the correct place thanks!
